en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-250-to-400.md

```diff
@@ -2979,22 +2979,8 @@ Follow the instructions below to move all the existing API Manager configuration
         ```
 
     - In order to work with the [API Security Audit Feature]({{base_path}}/design/api-security/configuring-api-security-audit/) you need to have the public certificate of the [42crunch](https://42crunch.com/) in the client-truststore. Follow the guidelines given in [Importing Certificates to the Truststore]({{base_path}}/install-and-setup/setup/security/configuring-keystores/keystore-basics/creating-new-keystores/#step-3-importing-certificates-to-the-truststore).
-
-5.  Configure the [SymmetricKeyInternalCryptoProvider](https://is.docs.wso2.com/en/5.11.0/administer/symmetric-overview/) as the default internal cryptor provider.
-    Generate your own secret key using a tool like OpenSSL.
-    
-    i.e.,
-       ```
-        openssl enc -nosalt -aes-128-cbc -k hello-world -P
-       ```        
-    Add the configuration to the <NEW_IS_HOME>/repository/conf/deployment.toml file.
     
-       ```
-       [encryption]
-       key = "<provide-your-key-here>"
-       ```
-
-6.  Upgrade the Identity component in WSO2 API Manager from version 5.6.0 to 5.11.0.
+5.  Upgrade the Identity component in WSO2 API Manager from version 5.6.0 to 5.11.0.
```

Comment:
We have set the default values for IS km deployment.toml

Reply:
fixed with https://github.com/wso2/docs-apim/pull/6044/commits/a2248a09bf63259f5a3e2b04ab018c34e608d92a